### ISSUE TITLE
Wrong in-memory API url in tutorial/toh-pt6.html

### ISSUE THREAD

**Issue by johncrim** [angular/angular.io#2883]
In https://angular.io/docs/ts/latest/tutorial/toh-pt6.html, under section "Heroes and HTTP", 2nd code example uses the wrong API url. The code is:

```
  private heroesUrl = 'app/heroes';  // URL to web api
```

and it should be:

```
  private heroesUrl = 'api/heroes';  // URL to web api
```

Also - at the end of that section, it says:

> Refresh the browser, and the hero data should be successfully loaded from the mock server.

This is incorrect, because the `HeroService.handleError()` fn has not been defined.

**Comment by wardbell** [comment]
While it _should_ be `'api/'`, in practice it doesn't matter because the in-memory-web-api actually doesn't care. Maybe it should, but it doesn't.  Of course it _would_ matter talking to a real web api where the base address would be `'api/'`,  

You're right also that we didn't show `handleError`;

Both problems will be fixed by PR #2899

**Comment by MickeySegal** [comment]
At https://github.com/angular/angular/issues/18869 I suggested making the documentation more clear on this and the wider issue.  It is unclear whether that will be done but at least it makes it easier to find this information by searching.
